plugins/connection/docker_api.py

```diff
@@ -12,7 +12,7 @@
 version_added: 1.1.0
 description:
     - Run commands or put/fetch files to an existing docker container.
-    - Uses Docker SDK for Python to interact directly with the Docker daemon instead of
+    - Uses the Requests library to interact directly with the Docker daemon instead of
```

Comment:
```suggestion
    - Uses the C(requests) library to interact directly with the Docker daemon instead of
```
this is totally opinion-based... feel free to use or ignore

Reply:
I made it lower-case, but added a link in 31eaf86fc6456a59192048db63b64d780474c8d2.